packages/collector/test/tracing/database/redis/test.js

```diff
@@ -235,6 +237,13 @@ const globalAgent = require('../../../globalAgent');
                       ]);
 
                       verifyHttpExit(controls, spans, writeEntrySpan);
+
+                      // TODO: not sure what is the difference in span length with setupType
```

Comment:
Why?

Reply:
This is from the main. we have to ignore it for now. Might change after the refactoring done in opt-in feature

Reply:
See https://github.com/instana/nodejs/blob/main/packages/collector/test/tracing/database/redis/test.js#L31

Reply:
Ah I see, the initial comment was confusing.